src/app/calendar/data.ts

```diff
@@ -0,0 +1,22 @@
+export enum ChangeMonthOrYear {
+  prevMonth = 1,
```

Comment:
Prvo veliko slovo, znaci PrevMonth. Ne moras da dodajes ostale vrednosti osim za prvi prop od enuma, on ce sam da dodeli vrednosti u zavisnosti od vrednosti prvog propertija,. Tipa ako je ovde PrevMonth(prvi prop)=1 onda ce NextMonth biti  jednak 2, PrevYear =3 itd. i to bez dodeljivanja vrednosti. Znaci ovako bi trebalo : PrevMonth=1, NextMonth, PrevYear, NextYear